src/server/controllers/pokemon.js

```diff
@@ -49,7 +49,7 @@ const createPokemons = async (req, res) => {
     }
     res.json("No Access")
 }
-
+/* This name needs changing and 1 is a magic number */
```

Comment:
Is it a magic number?

Reply:
This comment refers to `getPokemons`, which is a function used to fetch the pokemon ratings of a specific user. The name needs changing into something more explicit, and `profileId: 1` needs updated to a request body/param.

Reply:
I see. Yes that makes sense now!